coalib/bearlib/languages/definitions/Haskell.py

```diff
@@ -0,0 +1,16 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class Haskell:
+    extensions = ('.hs')
+    versions = 1.0, 1.1, 1.2, 1.3, 1.4
+    special_chars = '!#$%&*+./<=>?@\\^|-~'
+    comment_delimiters = '--',
+    multiline_comment_delimiters = {'{-': '-}'}
+    keywords = [
+        'case', 'class', 'data', 'default', 'deriving', 'do', 'else',
+        'forall', 'if', 'import', 'in', 'infix', 'infixl', 'infixr',
+        'let', 'module', 'newtype', 'of', 'qualified', 'then', 'type',
+        'where', '_', 'foreign', 'ccall', 'as', 'safe', 'instance', 'unsafe']
+    variable_declaration = {'::'}
```

Comment:
What is this field for? Btw `::` is not a variable declaration but type declaration.